src/charactersheet/models/dm/encounter_sections/encounter_item.js

```diff
@@ -40,6 +40,23 @@ export class EncounterItem extends KOModel {
     DESCRIPTION_MAX_LENGTH = 200;
     itemCurrencyDenominationOptions = Fixtures.general.currencyDenominationList;
 
+    totalCost = ko.pureComputed(() => {
+        if (this.quantity() &&
```

Comment:
not sure if I'm confusing this with work stuff, but don't we usually put the operator on the new line, like:
```
if (bool1
  && bool2
  && bool3
  .
 .. 
...) {
	...
}
```

Reply:
Fixed. 

FYI This was copy-pasted from `models/common/item.js`.